Multisig/UI/App/UIKit/UI Library/UIView+UINib.swift

```diff
@@ -32,6 +32,16 @@ extension UITableView {
         register(aClass.nib(), forCellReuseIdentifier: reuseID ?? aClass.reuseID)
     }
 
+    /// Registers a HeaderFooterView using a nib named the same as the view's class and
```

Comment:
This UITableView extension has become big enough to put it in a separate file. Let's extract?

Reply:
ok